pkgs/applications/networking/taler/default.nix

```diff
@@ -24,7 +24,7 @@ in rec {
       # "Server does not allow request for unadvertised object"
       deepClone = true;
```

Comment:
```suggestion
```

Reply:
Done. But what about the hash of taler-wallet-core? It changed too.

Reply:
I think they might’ve actually changed the tag in that case, whereas with exchange/merchant they didn’t, maybe it’s better to hardcode the revision

Reply:
Sorry but the build just fails when I try without the deepClone.

Reply:
Do you think we should wait until 0.9.4 to try fix this?

Reply:
I think we’re gonna keep having issues every time they push commits to that repo, so I don’t think we can wait till 0.9.4 to fix it

If you’re interested in investigating further you can set `fetchSubmodules = false;`, otherwise I’ll take a proper look at this when I’m at home later

Reply:
It doesn't build without the submodules. I don't know what to do here, sorry.